### ISSUE TITLE
Can POST a non-url and have it accepted

### ISSUE THREAD

**Issue by tbrown1979** [karan/s#9]
Example:

``` python
import requests

payload = {"long_url": "gibberish"}
r = requests.post('https://s-goel.herokuapp.com/new', data=payload)

print r.text
```

Response: 

``` json
{
  "status": "success",
  "short_id": "Fan",
  "short_url": "http://s-goel.herokuapp.com/Fan",
  "long_url": "gibberish"
}
```

Since 'gibberish' is stored as 'gibberish' when you try and go to 
http://s-goel.herokuapp.com/Fan
 you are rerouted to 
http://s-goel.herokuapp.com/gibberish

Upon visiting this url you're presented with: 

``` json
{
  "status": "success",
  "message": "id gibberish not found"
}
```

The front end checks that you're receiving a valid url, but if you do a normal POST you can bypass that security. The backend will also need to make sure it's receiving a url, I believe.

**Comment by karan** [comment]
Yup. Need to fix that, as in #6.

**Comment by karan** [comment]
The API assumes that the user passes a URL since the protocol, TLD, URL length can vary a lot, validation can never be correct. The JSON status has been fixed though!
